module/platform/Webots/src/Webots.cpp

```diff
@@ -260,18 +270,27 @@ void Webots::translate_and_emit_sensor(const SensorMeasurements& sensor_measurem
     for (const auto& position : sensor_measurements.position_sensors) {
         // string name
         // double value
+        translate_servo_id(position.name, sensor_data.servo).presentPosition = position.value;
     }
 
     for (const auto& accelerometer : sensor_measurements.accelerometers) {
         // string name
         // Vector3 value
+        sensor_data.accelerometer.x = static_cast<float>(accelerometers.value.X);
+        sensor_data.accelerometer.y = static_cast<float>(accelerometers.value.Y);
+        sensor_data.accelerometer.z = static_cast<float>(accelerometers.value.Z);
     }
 
     for (const auto& gyro : sensor_measurements.gyros) {
         // string name
         // Vector3 value
+        sensor_data.gyroscope.x = static_cast<float>(gyros.value.X);
+        sensor_data.gyroscope.y = static_cast<float>(gyros.value.Y);
+        sensor_data.gyroscope.z = static_cast<float>(gyros.value.Z);
```

Comment:
similar story here. I don't think `gyros` is defined, and we need to be assigning to the correct index, for the given `gyro`. As it stands both of these loops would (if `sensor_data.gyroscope` was a thing) just overwrite the same values for each loop iteration